apps/web/src/lang/fr.json

```diff
@@ -6806,12 +6782,28 @@
     "defaultMessage": "Nom du gestionnaire",
     "description": "Label for the user select field on team membership form"
   },
-  "hr/i9h": {
-    "defaultMessage": "Sélectionnez une équipe...",
-    "description": "Placeholder displayed for team selection input."
```

Comment:
this is new please add it back in

Reply:
dang, I'm going to blame turbo cache. Added back in [add lost translations back](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5921/commits/657b8652fd1d68848c65a3f5c5d5a879b4a536f4)